draft-packages/filter-menu-button/docs/FilterMenuButton.stories.tsx

```diff
@@ -242,3 +242,163 @@ export const DefaultWithChildrenAdvancedFilter = () => {
 DefaultWithChildrenAdvancedFilter.storyName =
   "Default with children (Advanced filter)"
 DefaultWithChildrenAdvancedFilter.parameters = { chromatic: { disable: false } }
+
+export const AutoHideBehaviours = () => {
+  const [isDropdownVisible1, setIsDropdownVisible1] = useState<boolean>(
+    IS_INITIAL_DROPDOWN_VISIBLE
+  )
+  const [isDropdownVisible2, setIsDropdownVisible2] = useState<boolean>(
+    IS_INITIAL_DROPDOWN_VISIBLE
+  )
+  const [isDropdownVisible3, setIsDropdownVisible3] = useState<boolean>(
+    IS_INITIAL_DROPDOWN_VISIBLE
+  )
+  const [appliedFilters, setAppliedFilters] = useState<DropdownOption[]>(
+    IS_CHROMATIC ? DROPDOWN_OPTIONS_CHROMATIC_SELECTED : []
```

Comment:
```suggestion
    []
```

We don't need to send this story to Chromatic (as mentioned in another comment)